pkg/metrics/instance/marshal_test.go

```diff
@@ -60,6 +60,7 @@ remote_write:
     max_backoff: 100ms
   follow_redirects: true
   metadata_config:
+    max_samples_per_send: 500
```

Comment:
This change worries me a little bit. Does one of our defaults need to change to re-sync with Prometheus or something?

Reply:
This new config param was added in 2.29.0 according to the prometheus change log. Here's the PR: https://github.com/prometheus/prometheus/pull/8959

Reply:
@rfratto looking into this closer, I'm not sure this is something we need to be worried it about. It seems previously there was no max size on the amount of metrics that could be sent in one shot, resulting in some rather large message sizes. this appears to be adding an upper bound on the max and500 is the default value. we should be able to adjust this in our configurations if necessary.

Reply:
Hmm, I think this called out to me because I remember documenting it before the docs rewrite where we link out to Prometheus. I think what happened here is we accidentally documented a field that didn't exist (for us) yet. 

Anyways, I'm not concerned anymore, we can leave this alone now that the mystery is solved :)